Code/GraphMol/Bond.h

```diff
@@ -98,8 +98,10 @@ class Bond : public RDProps {
     STEREOANY,       // intentionally unspecified
```

Comment:
Since you are using constructs like > STEREOANY, I would explicitly assign the values or at least STEREOANY.

```
STEREOANY=0,
```